mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx

```diff
@@ -2,22 +2,27 @@ import { ThemedScrollView } from '@components/themed'
 import { ThemedSectionTitle } from '@components/themed/ThemedSectionTitle'
 import { getAppLanguages, translate } from '@translations'
 import { RowLanguageItem } from '../components/RowLanguageItem'
+import { tailwind } from '@tailwind'
 
 export function LanguageSelectionScreen (): JSX.Element {
   const languages = getAppLanguages()
 
   return (
     <ThemedScrollView testID='language_selection_screen'>
```

Comment:
Should have a different background here. Please refer to the actual figma file for both light and Dark

Reply:
You can replace it with this component <ThemedScrollViewV2, automatically adds the default BG

Reply:
Done haha